turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksJavascriptInjector.java

```diff
@@ -0,0 +1,74 @@
+package com.basecamp.turbolinks;
+
+import android.content.Context;
+import android.util.Base64;
+import android.webkit.ValueCallback;
+import android.webkit.WebView;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class TurbolinksJavascriptInjector {
```

Comment:
I'm not sure this would really need its own class (yet). We've tried to keep the number of classes as tight as possible, as long as they don't get too big or stray too far from their intended uses.